docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Categorical/OneHotEncoding.cs

```diff
@@ -2,55 +2,65 @@
 using System.Collections.Generic;
 using Microsoft.ML;
 using Microsoft.ML.Data;
-using static Microsoft.ML.Transforms.OneHotEncodingEstimator;
+using Microsoft.ML.Transforms;
 
-namespace Samples.Dynamic
+namespace Samples.Dynamic.Transforms.Categorical
```

Comment:
>.Transforms.Categorical [](start = 25, length = 23)

let's not worry about adding to the namespace. we only did it for trainers, b/c we have identical names for multiple ML tasks, so we used namespace to avoid class name conflict. for transforms we don't have that problem. we can live with the simple namespace.

Reply:
I will prefer way this to keep things consistent and to indicate this is a transform.

---
In reply to: [278764268](https://github.com/dotnet/machinelearning/pull/3588#discussion_r278764268) [](ancestors = 278764268)